src/test/java/io/vlingo/lattice/grid/hashring/MurmurSortedMapHashRingTest.java

```diff
@@ -0,0 +1,46 @@
+package io.vlingo.lattice.grid.hashring;
+
+import org.junit.Test;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.junit.Assert.assertEquals;
+
+public class MurmurSortedMapHashRingTest {
+
+  @Test
+  public void testNodeOfRace() {
```

Comment:
Should this be integrated into the "property" test for all HashRing implementations?

Reply:
I would focus on the `HashRing` implementation that we will use. Perhaps we should even delete the others or at least mark them as deprecated. I don't like throwing away potentially useful work, but if murmur works best what's the use of retaining the others? At least they should be deprecated.